boards/arm/mec15xxevb_assy6853/mec15xxevb_assy6853_defconfig

```diff
@@ -14,3 +14,11 @@ CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=48000000
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
+
+CONFIG_MULTI_LEVEL_INTERRUPTS=y
+CONFIG_2ND_LEVEL_INTERRUPTS=y
+CONFIG_MAX_IRQ_PER_AGGREGATOR=32
+CONFIG_NUM_2ND_LEVEL_AGGREGATORS=18
```

Comment:
This is your SoC so it's your call, but why is this 18 if only 9 aggregators are being added to the dtsi? This seems wasteful of flash space in `_sw_isr_table`, especially since it looks like the DT yaml bindings already provide a way to map a level 1 interrupt to a girq number. Am I missing something?

Reply:
Answer is the crazy HW design.  All the interrupts by default are aggregated but customers wanted more control over priority of individual interrupt sources. HW provided alternate routing of most sources to additional NVIC inputs. MEC15xx has 174 NVIC inputs, the first 18 are the aggregated GIRQ's. The remaining NVIC inputs are status from a subset of the aggregators. Unfortunately, the sources that can be connected directly to the NVIC are not limited to last 4 or 5 aggregators as in openisa_rv32m1 but are sparse.  I feel I need to change the python interrupt table generator to handle sparse tables that need to be condensed into a smaller memory foot print.  Or is it better to create a custom python table generator for this SoC?

Reply:
Thanks for your responses. I'll try to build an application for your board using this feature and look at how it lays out in the build directory to see if I can understand what you mean, and will follow up if I'm still confused.